.github/workflows/cypress.yaml

```diff
@@ -41,6 +41,8 @@ jobs:
         run: "poetry run python deploy/cypress_setup.py -p opscenter"
       - name: run streamlit
         run: "poetry run streamlit run app/ui/Home.py &"
+      - name: Wait one minute for materialization to happen
+        run: sleep 60
```

Comment:
In the same way that we codified some setup in `cypress_setup.py`, is it possible we check for a discrete condition via SQL objects rather than a blind wait?

It looks like the `INTERNAL.TASK_QUERY_HISTORY` and `INTERNAL.TASK_WAREHOUSE_EVENTS` tables might already contain the information we'd need.

Reply:
I opened #47 to see if that is possible, but we could try to do that now.  How would we know exactly that the data is there, look at the count?